packages/picasso-rich-text-editor/src/LexicalEditor/utils/cleanupHtmlOutput.test.ts

```diff
@@ -0,0 +1,47 @@
+import { cleanupHtmlOutput } from './cleanupHtmlOutput'
+
+// Test cases structure:
+// removeWhat (string), removeFrom (string), input (string), expected (string)
+const tests = [
```

Comment:
As comments say these are testCases, not tests
```suggestion
const testCases = [
```

Reply:
@OleksandrNechai Fixed, thanks!